app/views/documents/show/_requirements.html.erb

```diff
@@ -1,13 +1,15 @@
 <% issue_params = {
   style: "summary",
-  link_options: {
-    title: { href: "#content" },
-    summary: { href: "#content" },
-    body: { href: "#content" },
-    change_note: { href: "#content" },
-    alt_text: { href: "#lead-image" },
-    caption: { href: "#lead-image" },
-    topics: { href: "#topics" }
+  hrefs: {
+    title: edit_document_path(anchor: "title"),
+    summary: edit_document_path(anchor: "summary"),
+    body: edit_document_path(anchor: "body"),
+    change_note: edit_document_path(anchor: "changenote"),
+    #Come back to this at some point so we could link directly to the image?
+    alt_text: images_path(@edition.document, anchor:"imagelist"),
+    caption: images_path(@edition.document, anchor:"imagelist"),
+    #--
```

Comment:
I'd have this just remain linking to lead image until we have a strategy for these, rather needing to leave a todo comment. 

We do have precedent for leaving TODO and FIXME comments, so if you do need to do them it should follow the existing format: https://github.com/alphagov/content-publisher/blob/04e2640b0092f930e058c1502a1a83a900b436af/app/controllers/editions_controller.rb#L7-L8 but in general we discourage them.

Also minor but you missed space characters after the colon of the hash in both of those.